spec/faraday/adapter/em_synchrony_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+RSpec.describe Faraday::Adapter::EMSynchrony do
+  features :request_body_on_query_methods, :reason_phrase_parse, :skip_response_body_on_head, :parallel
```

Comment:
Perhaps this is a too-long line?

Reply:
Decided no to address it here as we still have to decide the project-wise max line length.
I personally think the default 80 characters is a bit too restrictive and a bit 80's, but we'll see once we decide to take down that cop.